script1.js

```diff
@@ -0,0 +1,26 @@
+var cssInput = document.querySelector('#cssInput');
+cssInput.addEventListener('input', (e) => {
+  styleParser(e.target.value);
+})
+function styleParser(text) {
+  //break up into sections based on ending curlybrackets
+  var splitText = text.split('}')
+  for (let i = 0; i < splitText.length; i++) {
```

Comment:
Why did you choose to do a For Loop instead of a ForEach Loop here?

Reply:
Both accomplish the same thing.  It was likely a stylistic coding choice.  ForEach does allow access to the index as the second parameter to the forEach callback, but I don't think it would have provided any extra value to use forEach.

Reply:
Mostly because I have 2 levels of nested loops and nesting a forEach inside a forEach inside a forEach would have been ugly.